app/defaultWalletNodes.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Could we please remove this file from the check in? We have added it to .gitignore and, for legal, reasons we don't add it to the repo.